publish.js

```diff
@@ -178,41 +184,11 @@ function patchMetadata (current, newData, opts) {
         }
         break
 
-      // ignore these
```

Comment:
does this mean that we can put `maintainers` in the package.json and it'll try to copy over the root packument?

(If so.... that's not good!)

Reply:
I'm good with removing the maintainers from the code paths where we _create_ a maintainers entry in the package.json manifest, but if they user had put one in the package.json file themselves (likely without realizing what it is), we do need to skip over it in this part of the code where we're copying things to the root packument.

So I think my only change would be to bring this one part of the diff back.

Reply:
Nevermind!  Yes, I was misreading this, sorry for the confusing back and forth.  Had to actually look at the code rather than the diff to sort it out.  This is fine as you have it written here :)